string label
geometry_msgs/Point center  # in pixels, from center (haha, maybe not really)
geometry_msgs/Vector3 size  # in pixels
